src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    private static int NUMBER_INDEX = 100;
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public String getRandomBall() {
+
+        Ball randomBall = new Ball();
+        String color = randomBall.getColor();
+        color = randomColor.getRandomColor();
+        Random random = new Random();
+        int number = randomBall.getNumber();
+        number = random.nextInt(NUMBER_INDEX) + 1;
+        String numberToString = Integer.toString(number);
```

Comment:
no need to explicitly cast an Integer to a String.
when you make the operation `concatenation` all objects automatically will be cast to the string by toString() method.